libs/vscode-ui/feature-task-execution-form/src/lib/task-execution-form.component.spec.ts

```diff
@@ -5,6 +5,8 @@ import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';
 import { TaskExecutionSchema } from '@nx-console/schema';
 import { TASK_EXECUTION_SCHEMA } from './task-execution-form.schema';
 import { VscodeUiComponentsModule } from '@nx-console/vscode-ui/components';
```

Comment:
I can't figure out why VS Code shows this lint error on these [ArgumentListModule imports](https://github.com/nrwl/nx-console/pull/1059/files#diff-37a9daab57c623d442d8f82f28baadfdd1b1db33e7c77838aa99c82b2c92237aR8). It does't fail `nx lint` and another weird things is VS Code allows it if I change it to a relative import (which doesn't pass lint obvs but wtf).

Also the vscode-ui project is in apps and nothing is being imported from it here. Kinda feels like the partial matching on this rule isn't right.
```
Circular dependency between "vscode-ui-feature-task-execution-form" and "vscode-ui" detected: vscode-ui-feature-task-execution-form -> vscode-ui -> vscode-ui-feature-task-execution-formeslint@nrwl/nx/enforce-module-boundaries

Reply:
Whatever was causing this went away, so cool 😎